tests/sentry/sentry_metrics/test_batch.py

```diff
@@ -122,6 +136,60 @@ def _get_string_indexer_log_records(caplog):
     ]
 
 
+@pytest.mark.parametrize(
+    "use_case_key, should_index_tag_values, expected",
+    [
+        pytest.param(
+            UseCaseKey.PERFORMANCE,
```

Comment:
`use_case_key` is the same all the time so technically it doesn't have to be a pytest param